projects/frontend/data-pipelines/gui/projects/data-pipelines/src/lib/components/data-jobs-explore/components/grid/data-jobs-explore-grid.component.spec.ts

```diff
@@ -333,10 +344,14 @@ describe('DataJobsExploreGridComponent', () => {
     });
 
     describe('search', () => {
+        beforeEach(() => {
+            fixture.detectChanges();
+        });
+
         it('makes expected calls', () => {
             spyOn(component, 'loadDataWithState').and.callThrough();
             component.search('searchValue');
-            expect(component.searchQueryValue).toBe('searchValue');
+            expect(component.clrGridUIState.search).toBe('searchValue');
```

Comment:
I see many magic string literals in the PR, is it good idea to be defined as constants?

Reply:
They are numbers to acquire specific intercepted request from the framework and I don't agree with the static analyzer.

e.g.
```
expect(setQueryParamSpy.calls.argsFor(11)).toEqual([QUERY_PARAM_SEARCH, 'search test value 2', 0]);
```

with `argsFor(11)` we will get parameters for call with id number 11